<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lecturer_Home_Auth</class>
 <widget class="QMainWindow" name="Lecturer_Home_Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lecturer Home Auth</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/assets/presence_logo.png</normaloff>:/res/assets/presence_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget #centralwidget {
	background-color: #FFFFFF;
}</string>
   </property>
   <widget class="QGroupBox" name="headerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="headerHud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/header_large.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/back.png</normaloff>:/res/assets/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="headerWelcomeLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFF;
	font-family: Poppins;
	font-size: 20px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Hello, &lt;?&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>Light</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 300;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: right;
	font-family: Poppins;
	font-size: 11px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerAvatarLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/cict_avatar.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerCollegeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>C&lt;?&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="headerLecturerIdLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>LN: ****</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>360</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="footerHud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/footer.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>17</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/home_selected.png</normaloff>:/res/assets/home_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="homeLabel">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>42</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="classListButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>17</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/class_list_unselected.png</normaloff>:/res/assets/class_list_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="classListLabel">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>42</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>CLASS LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="authInfoGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>360</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="presenceIcon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>320</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/presence_logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="schoolLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>240</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000F2A;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>West Visayas State University</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	text-align: center;
	font-family: Poppins;
	font-size: 20px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>LN&lt;?&gt;, FN&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lecturerIdLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	font-family: Poppins;
	font-size: 16px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>LN: ****</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="collegeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>320</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #19283F;
	font-family: Poppins;
	font-size: 14px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>C&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>10</y>
       <width>310</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background: none;
	color: #FF003D;
	font-family: Poppins;
	font-size: 10px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="authTitleLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>320</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000000;
	text-align: center;
	font-family: Poppins;
	font-size: 20px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>Auth Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="authCodeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>320</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #000000;
	text-align: center;
	font-family: Poppins;
	font-size: 48px;
	font-style: normal;
	font-weight: 700;
	line-height: normal;
	letter-spacing: 4.8px;
}</string>
     </property>
     <property name="text">
      <string>######</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="qrCodeButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/qr_code_unselected.png</normaloff>:/res/assets/qr_code_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="authCodeButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>400</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}

QPushButton:focus { 
	outline: none; 
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/auth_code_selected.png</normaloff>:/res/assets/auth_code_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
